docs/sources/tempo/operations/dedicated_columns.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Dedicated columns
+description: Learn how to use dedicated columns to improve query performance.
+weight: 42
+---
+
+# Dedicated columns
+
+{{% admonition type="warning" %}}
+Dedicated columns is part of Parquetv3, an [experimental feature](/docs/release-life-cycle/). Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided.  
+{{% /admonition %}}
+
+Dedicated columns improve query performance by storing the most frequently used columns in dedicated columns,
+rather than in the generic attribute key-value list.
+
+Introduced with `vParquet3`, dedicated columns are only available when using this storage format.
+To read more about the design of `vParquet3`, see [the design proposal](https://github.com/grafana/tempo/blob/main/docs/design-proposals/2023-05%20vParquet3.md).
+
+## Configuration
+
+Dedicated columns can be configured in the storage block or via overrides.
+
+```yaml
+# Storage configuration for traces
+storage:
+  trace:
+    block:
+      # Default dedicated columns for all blocks
+      dedicated_columns:
+        - name: <string>, # name of the attribute
+          type: <string>, # type of the attribute. options: string
+          scope: <string> # scope of the attribute. options: resource, span
+
+overrides:
+  # Global overrides for dedicated columns configuration
+  dedicated_columns:
+    - name: <string>, # name of the attribute
+      type: <string>, # type of the attribute. options: string
+      scope: <string> # scope of the attribute. options: resource, span
+
+  per_tenant_override_config: /conf/overrides.yaml
+---
+# /conf/overrides.yaml
+# Tenant-specific overrides configuration
+overrides:
+  "<tenant id>":
+    dedicated_columns:
+      - name: <string>, # name of the attribute
+        type: <string>, # type of the attribute. options: string
+        scope: <string> # scope of the attribute. options: resource, span
+
+  # A "wildcard" override can be used that will apply to all tenants if a match is not found.
+  "*":
+    dedicated_columns:
+      - name: <string>, # name of the attribute
+        type: <string>, # type of the attribute. options: string
+        scope: <string> # scope of the attribute. options: resource, span
+```
+
+Priority is given to the most specific configuration, so tenant-specific overrides will take precedence over global overrides.
+Similarly, default overrides take precedence over storage block configuration.
+
+## Usage
+
+Dedicated columns are limited to 10 span attributes and 10 resource attributes.
```

Comment:
I think it would be good to emphasize that only strings are supported.
```suggestion
Dedicated columns are limited to 10 span attributes and 10 resource attributes with string values.
```